install.sh

```diff
@@ -617,7 +617,9 @@ getshims() {
 killtree $({ set +x; } 2>/dev/null; getshims; set -x)
 
 do_unmount_and_remove() {
-    awk -v path="$1" '$2 ~ ("^" path) { print $2 }' /proc/self/mounts | sort -r | xargs -r -t -n 1 sh -c 'umount "$0" && rm -rf "$0"'
+    while read -r _ path _; do
```

Comment:
can you wrap this in a `$({ set +x; } 2>/dev/null;   set -x;)` as seen above? The use of a shell loop to filter the paths results in running this uninstall script filling the terminal with:
```console
+ read -r _ path _
+ case "$path" in
+ read -r _ path _
+ case "$path" in
+ read -r _ path _
+ case "$path" in
+ read -r _ path _
+ case "$path" in
+ read -r _ path _
```